folly/dynamic-inl.h

```diff
@@ -134,17 +134,11 @@ struct FOLLY_EXPORT TypeError : std::runtime_error {
       const std::string& expected,
       dynamic::Type actual1,
       dynamic::Type actual2);
-  // TODO: noexcept calculation required through gcc-v4.9; remove once upgrading
-  // to gcc-v5.
-  TypeError(const TypeError&) noexcept(
-      std::is_nothrow_copy_constructible<std::runtime_error>::value);
-  TypeError& operator=(const TypeError&) noexcept(
-      std::is_nothrow_copy_assignable<std::runtime_error>::value);
-  TypeError(TypeError&&) noexcept(
-      std::is_nothrow_move_constructible<std::runtime_error>::value);
-  TypeError& operator=(TypeError&&) noexcept(
-      std::is_nothrow_move_assignable<std::runtime_error>::value);
-  ~TypeError() override;
+  TypeError(const TypeError&) = default;
+  TypeError& operator=(const TypeError&) = default;
+  TypeError(TypeError&&) = default;;
+  TypeError& operator=(TypeError&&) = default;
+  ~TypeError() override = default;
```

Comment:
Actually can we just omit all five of these and let them all be implicitly defaulted?

Reply:
Yep, even better. The special member functions will be implicitly generated and be noexcept-correct. 👍 

Just pushed up a change to just omit them all.